### ISSUE TITLE
Goyo enter and exit does not remove tmux

### ISSUE THREAD

**Issue by vmuthuk2** [junegunn/goyo.vim#271]
I was using below code to hide tmux while inside Goyo which was working until I recently updated neovim to v0.8.0. Is there a fix for this?
```vim
function! s:goyo_enter()
  if executable('tmux') && strlen($TMUX)
    silent !tmux set status off
    silent !tmux list-panes -F '\#F' | grep -q Z || tmux resize-pane -Z
  endif
  set noshowmode
  set noshowcmd
  set scrolloff=999
  Limelight
  " ...
endfunction
```

**Comment by junegunn** [comment]
Did you setup `autocmd`s?

**Comment by vmuthuk2** [comment]
Yes, I use `autocmd! User GoyoEnter nested call <SID>goyo_enter()`. I have this issue on both Mac and Linux recently.

**Comment by junegunn** [comment]
What about the other lines of code in the function? (set ..., Limelight) Are they executed as expected?

**Comment by vmuthuk2** [comment]
Sorry tmux is hidden inside Goyo but the lualine and tab bar of neovim is not hidden. Refer to figure below:
<img width="1260" alt="image" src="https://user-images.githubusercontent.com/25335803/189549711-1ae3d5d7-4741-4851-8a0c-2bdfa8f45ad6.png">

**Comment by junegunn** [comment]
Goyo doesn't know what lualine is. I can't update Goyo whenever a new statusline plugin comes into being. You have to call its API for disabling/enabling it on `GoyoEnter` and `GoyoLeave`.
